### PR TITLE
Reconfigure live system

### PR DESCRIPTION
**Pull request by mgax**
Adds `reconfigure` tags to a bunch of tasks. Also adds support for liquid-core to re-run ansible.

### CODE REVIEW

File: ansible/roles/liquid-core/templates/local.py [hunk 1]
```diff
@@ -20,4 +20,7 @@
 LIQUID_DOMAIN = '{{ liquid_domain }}'
 DISCOVERY_URL = 'http://localhost:13777'
 
+LIQUID_SETUP_RECONFIGURE = 'sudo /opt/setup/libexec/liquid-core-reconfigure'
```
Comment (gabriel-v) [inline ansible/roles/liquid-core/templates/local.py]: why not use the `bin` folder which is already there?

File: py/liquid/boot.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+        else:
+            print("First boot done.")
+            first_boot_done.touch()
+
+    else:
+        print("Not starting first boot, already done.")
+
+
+    print("Starting all services.")
```
Comment (gabriel-v) [inline py/liquid/boot.py]: There was a `set +e` around here. We don't want only some of the on-boot scripts to run if others fail.

File: ansible/.gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 /vars/config.yml
+/vars/liquidcore.yml
```
Comment (gabriel-v) [inline ansible/.gitignore]: Is this added in the ansible playbooks? `server.yml` `board_chroot.yml`

**Comment by gabriel-v** [comment]
closes https://github.com/liquidinvestigations/liquidinvestigations/issues/80

**Comment by mgax** [comment]
Merged by hand.

**Pull request closed without merging**